### PR TITLE
JESI-2333

### PR DESCRIPTION
**Pull request by AndreTheHunter**
[JESI-2333](https://jesims.atlassian.net/browse/JESI-2333)

### CODE REVIEW

File: src/io/jesi/backpack/miscellaneous.cljc [hunk 1]
```diff
@@ -70,3 +71,12 @@
     :default sym))
 
 ;TODO create a macro that take a form and can transform it to the required runtime e.g. cljs ns rename, cljs catch clause
+
+(defn collify
+  "Puts value `v` in a vector if it is not a collection. Returns `nil` if no value"
+  ([] nil)
+  ([v]
+   (condp call v
+     nil? nil
```
Comment (AndreTheHunter) [inline src/io/jesi/backpack/miscellaneous.cljc]: Not 100% convinced that this should return `nil`, but `(= nil (seq []))`. so everything that takes a collection _should_ handle `nil`

File: package-dry.json [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
+  "license": "EPL-1.0",
+  "bugs": {
+    "url": "https://github.com/jesims/backpack/issues"
+  },
+  "devDependencies": {
+    "standard": "^12.0.1",
+    "karma": "^4.0.0",
+    "karma-chrome-launcher": "2.2.0",
+    "karma-cljs-test": "0.1.0",
+    "karma-clear-screen-reporter": "^1.0.0"
```
Comment (axrs) [inline package-dry.json]: Why bother putting versions if managed above?

File: package-dry.json [hunk 3]
```diff
@@ -1,25 +1,34 @@
...
+  },
   "dry": {
     "extends": "@io.jesi/parent/package-dry.json",
     "dependencies": {
       "@io.jesi/parent": "0.0.7"
     }
```
Comment (axrs) [inline package-dry.json]: The parent was the one giving the versions. If we're just setting them. Lets remove parent.

File: src/io/jesi/backpack/miscellaneous.cljc (hunk 1 above)
Comment (axrs) [reply src/io/jesi/backpack/miscellaneous.cljc]: Give nil get nil

File: package-dry.json (hunk 3 above)
Comment (AndreTheHunter) [reply package-dry.json]: `dry install --save-dev karma-clear-screen-reporter` changed the file. Fixed.

File: package-dry.json (hunk 2 above)
Comment (AndreTheHunter) [reply package-dry.json]: `dry install --save-dev karma-clear-screen-reporter` changed the file. Fixed.

**Pull request merged**